src/App.tsx

```diff
@@ -222,14 +222,19 @@ const Profile = ({ user, userData, userName }) => {
   }
   let fullName = user && user.displayName;
   let youtubeUrl = null;
+  let trustLevel = 0;
+  let networkTime = 0;
   if (userData) {
     userName = userName || userData.get('user_name');
     fullName = userData.get('full_name');
     youtubeUrl = userData.get('invite_url');
+    trustLevel = 3; // TODO Hard coded need access to firebase
+    networkTime = 30;
   }
   return (
     <div>
       <div>{fullName}</div>
+      {trustLevel && <TrustLevel trustLevel={trustLevel} networkTime={networkTime}/>}
```

Comment:
Wanna do something like `user && trustLevel` instead of only `trustLevel` so that people only see this at `/me` ... also maybe we should refactor or rename some variables, that's not exactly an intuitive fix :p